test/react-web/client/Query.test.js

```diff
@@ -35,6 +35,10 @@ const catchAsyncError = (done, cb) => {
 };
 
 describe('Query component', () => {
+  beforeEach(() => {
+    jest.useRealTimers();
```

Comment:
If we don't reset this and one of our test fails this will cause other tests to incorrectly fail due to timeouts. This seems to be the safest way to ensure that other tests are unaffected.